src/Time/Units.hs

```diff
@@ -162,11 +176,19 @@ instance KnownUnitName unit => Read (Time unit) where
         readP :: ReadP (Time unit)
         readP = do
             let naturalP = read <$> munch1 isDigit
-            n <- naturalP
-            m <- option 1 (char '/' *> naturalP)
+            -- If a '+' is parsed as part of a mixed fraction, the other parts
+            -- are no longer optional.  This separation is required to prevent
+            -- e.g. "3+2" successfully parsing.
```

Comment:
I still can write `3+2/1s` and it will be parsed as `5s` :stuck_out_tongue_winking_eye:. So I think at this moment we need to decide whether we should also forbid using `1` in the denominator, because it's kinda similar cases for me..
What are your thoughts @ChShersh

Reply:
I don't know really... Tough decisions 🤔

Reply:
Hmm, true, and I don't enforce correct separation of mixed fractions either.  This could be fixed by doing that (so that the remainder part has to have numerator < denominator)

Reply:
Yes, true.. 
Let's keep it as it is now for the moment :slightly_smiling_face: